installation.rst

```diff
@@ -119,6 +119,8 @@ If running from a jupyter console, either the ``jupyter-qtconsole``, the ``jupyt
 the console within `Spyder <https://pythonhosted.org/spyder/>`_, you may need to ensure a few other
 `IPython magic <https://ipython.org/ipython-doc/3/interactive/tutorial.html#magic-functions>`_
 functions are called prior to using vispy in a given kernel.
+To use the ``%pylab`` magici command, you will need to install ``matplotlib`` using your favorite
+method for installing python packages.
```

Comment:
Could you try removing the pylab call and see if it still works? I doubt it is needed, but I had always done both. If it isn't needed then you could remove it from the instructions.